src/main.cpp

```diff
@@ -92,7 +94,6 @@ int connectmode = CONNECTMODE;
 
 int offlineMode = 0;
 const int OnlyPID = ONLYPID;
-const int TempSensor = TEMPSENSOR;
```

Comment:
as far as I know we need the TEMPSENSOR constant macro as a int in the code, so we can change it thru the webseite later